life_expectancy/tests/test_main.py

```diff
@@ -0,0 +1,27 @@
+"""Tests for the cleaning module"""
+import pandas as pd
+from pytest import MonkeyPatch
+from life_expectancy.main import main
+from . import FIXTURES_DIR
+
+
+def test_main(monkeypatch: MonkeyPatch, pt_life_expectancy_expected_test):
+    """Run the `clean_data` function and compare the output to the expected output"""
+
+    monkeypatch.setattr( "life_expectancy.loading_saving.save_data",
+                        lambda _ : print('MOCK SAVE')
+                        )
+    monkeypatch.setattr( "life_expectancy.loading_saving.load_data",
+                        lambda  : pd.read_csv(FIXTURES_DIR/
+                                              'eu_life_expectancy_loaded_dataframe_test.csv')
+                        )
+
+    pt_life_expectancy_obtained_test = main(
+        country='PT',
+        input_file_path=FIXTURES_DIR / "eu_life_expectancy_raw_test.tsv",
+        output_file_path=FIXTURES_DIR / 'pt_life_expectancy_obtained_test.csv'
+    )
+
+    pd.testing.assert_frame_equal(
```

Comment:
Just my personal taste here, but I would add a assert to check if the save data is being called. Fine either way, since you already test save data in another function